hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/DefaultStoreFileManager.java

```diff
@@ -41,7 +45,7 @@
  * Default implementation of StoreFileManager. Not thread-safe.
  */
 @InterfaceAudience.Private
-class DefaultStoreFileManager implements StoreFileManager {
+public class DefaultStoreFileManager implements StoreFileManager {
```

Comment:
Why we need to make this public?

Reply:
Adding dual file writing capability to default compactor but not mob compaction required MobStoreEngine not to extend DefaultStoreEngine anymore. Instead MobStoreEngine now extends StoreEngine<DefaultStoreFlusher, RatioBasedCompactionPolicy, DefaultMobStoreCompactor, DefaultStoreFileManager>. This required making DefaultStoreFileManager public.